coordinator/tests/services/validation/bundle-validator.spec.ts

```diff
@@ -388,6 +388,17 @@ describe("MedicationRequest consistency checks", () => {
       errors.medicationRequestDuplicateIdentifierIssue
     )
   })
+
+  test("Should throw error when PrescriptionType is 01nn and healthcareService exists and partOf doesn't", () => {
+    const prescriptionTypeExtension = getExtensionForUrlOrNull(
+      medicationRequests[0].extension,
+      "https://fhir.nhs.uk/StructureDefinition/Extension-DM-PrescriptionType",
+      'Entry("MedicationRequest").extension("https://fhir.nhs.uk/StructureDefinition/Extension-DM-PrescriptionType")',
+    ) as fhir.CodingExtension
+
+    prescriptionTypeExtension.valueCoding.code = "0101"
+  })
```

Comment:
- [ ] Should this test use `verifyPrescriptionBundle` and check what errors are returned?
- [ ] Do we have another test for secondary care prescriptions?

Reply:
Realised I came across the issue with test packs midway through writing tests, finished up those tests now and added